### PR TITLE
Added prettier and eslint and formatted/linted code according to them

### PR DESCRIPTION
**Pull request by Takkes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,20 @@
...
+    "eslint": "^7.4.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.3.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.3",
+    "eslint-plugin-react-hooks": "^4.0.0"
```
Comment (inergy) [inline package.json:39]: are all of these plugins required due to peer deps or something? If not, I'd suggest limited to only the ones needed as to not be misleading that they are in use

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+	"plugins": ["prettier"],
+    "rules": {
+		"prettier/prettier": ["error"],
+		"no-param-reassign": ["error", { "props": false }],
+		"no-underscore-dangle": "off",
+		"no-plusplus": "off",
+		"class-methods-use-this": "off"
+    }
+}
```
Comment (inergy) [inline .eslintrc.json:21]: maybe use 2 space indentation to keep things consistent

File: package.json (hunk 1 above)
Comment (Takkes) [reply package.json:39]: At least I did not install them myself. I only installed prettier, eslint, eslint-config-prettier and eslint-plugin-prettier. The rest got installed automatically. I don't know why react is in there though or how to disable it.

File: .eslintrc.json (hunk 2 above)
Comment (Takkes) [reply .eslintrc.json:21]: Yeah, looks like tabs and spaces got mixed up. Will fix that

**Comment by Takkes** [comment]
Will create new PR with new structure

**Pull request closed without merging**